libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -467,58 +464,78 @@ pub fn find_timing_mark_grid(
 ///      └──────────┘
 ///
 /// ```
-#[allow(clippy::too_many_lines)]
-fn score_timing_mark_at_rect(ballot_image: &BallotImage, rect: &Rect) -> Option<UnitIntervalScore> {
+fn score_timing_mark_geometry_match(
+    ballot_image: &BallotImage,
+    timing_mark: &Rect,
+    geometry: &Geometry,
+) -> TimingMarkScore {
     let image = &ballot_image.image;
     let threshold = ballot_image.threshold;
-    let rect_height = rect.height() as i32;
     let image_rect = Rect::new(0, 0, image.width(), image.height());
+    let expected_width = geometry.timing_mark_size.width as PixelUnit;
+    let expected_height = geometry.timing_mark_size.height as PixelUnit;
+    let expected_timing_mark_rect = Rect::new(
+        timing_mark.left(),
+        timing_mark.top(),
+        expected_width,
+        expected_height,
+    );
     let search_rect = Rect::new(
-        rect.left() - rect_height,
-        rect.top() - rect_height,
-        rect.width() + 2 * rect.height(),
-        rect.height() + 2 * rect.height(),
-    )
-    .intersect(&image_rect)?;
-    let black_pixel_search_area = rect.width() * rect.height();
+        timing_mark.left() - expected_height as i32,
+        timing_mark.top() - expected_height as i32,
+        expected_width + 2 * expected_height,
+        3 * expected_height,
+    );
```

Comment:
It would likely decrease the mark scores for some side timing marks. I'm not sure off-hand that it would be better or worse, but we probably would have to adjust the score thresholds.

Reply:
Wouldn't we want it to be balanced for left and right side timing marks though?

Reply:
That's a fair point. Right now any cropped left-hand timing marks are likely to have a higher padding score than similar right side marks. I'll change it.